src/trivia/util.h

```diff
@@ -270,8 +270,12 @@ alloc_failure(const char *filename, int line, size_t size)
  * including padding if any.
  */
 #ifndef offsetof
-#define offsetof(type, member) ((size_t) &((type *)0)->member)
-#endif
+#  if defined(__has_builtin) && __has_builtin(__builtin_offsetof)
```

Comment:
`__has_builtin` is defined a few lines above so `defined(__has_builtin)` could be dropped:

https://github.com/tarantool/tarantool/blob/b0b32bff022caeffd64ca687baab908f25fa2fcb/src/trivia/util.h#L177-L185

Reply:
Forgot about that. Thanks, I removed that check.